colossus-tests/src/test/scala/colossus/core/ServerSpec.scala

```diff
@@ -308,6 +308,26 @@ class ServerSpec extends ColossusSpec {
         alive() must equal(0)
 
       }
+
+      "attempt to re-register connection if refused by worker" taggedAs(org.scalatest.Tag("test")) in {
```

Comment:
No I throw it in when I want to just run one test but I keep forgetting to remove it afterwards :ghost: